pymc/aesaraf.py

```diff
@@ -1015,3 +1019,144 @@ def compile_pymc(
         **kwargs,
     )
     return aesara_function
+
+
+def compile_forward_sampling_function(
```

Comment:
Any preference between `forward_sampling` or `ancestral_sampling`?

Reply:
Forward sampling all the way. Ancestral sampling seems like some black magic horoscope thing